### PR TITLE
Fix: Omit sponsors sync from changelog

### PR DESCRIPTION
**Pull request by nzakas**

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 env:
   node: true
-extends: "eslint-config-eslint"
+# extends: "eslint-config-eslint"
```
[the remark this answers is not in the document]
Comment (nzakas) [reply .eslintrc.yml:3]: Oops, forgot to rebase this after the merge.

**Comment by nzakas** [comment]
Should be all set now.

File: .eslintrc.yml (hunk 1 above)
Comment (aladdin-add) [inline .eslintrc.yml:3]:
```suggestion
extends: "eslint-config-eslint"
```

Comment (nzakas) [reply .eslintrc.yml:3]: Thanks!

**Review by aladdin-add — APPROVED** [review]
LGTM!

**Review by mdjermanovic — APPROVED** [review]
LGTM!

**Pull request merged**